README.md

```diff
@@ -47,7 +47,7 @@ In other words, it's possible to have e.g. one sublevel with Buffer keys and val
 The optional `opts` parameter has the following `subleveldown` specific properties:
 
 * `opts.separator` *(string, default: `'!'`)* Character for separating sublevels
-* `opts.open` *(function)* Optional open hook called when the underlying `levelup` instance has been opened
+* `opts.open` *(function)* Optional open hook called when the underlying `levelup` instance has been opened. The hook has the signature `hook(cb)`, where `cb` is the callback passed to `SubDb#open`.
```

Comment:
"where `cb` is the callback passed to `SubDb#open`" is an implementation detail. Perhaps say something like "the callback must be called to finish opening" instead.